core/templates/domain/learner_dashboard/learner-dashboard-activity-ids.model.ts

```diff
@@ -91,6 +105,22 @@ export class LearnerDashboardActivityIds {
     }
   }
 
+  belongsToCompletedStories(storyId: string): boolean {
+    if (this.completedStoryIds.indexOf(storyId) !== -1) {
+      return true;
+    } else {
```

Comment:
Ditto here and elsewhere on the else statement. You can directly return false after the first check.

Reply:
Done.